src/webgpu/api/operation/sampling/anisotropy.spec.ts

```diff
@@ -270,7 +272,7 @@ g.test('anisotropic_filter_mipmap_color')
       maxAnisotropy: 1,
       _results: [
         { coord: { x: xMiddle, y: 2 }, expected: colors[2] },
-        { coord: { x: xMiddle, y: 6 }, expected: [colors[0], colors[1]] },
+        { coord: { x: xMiddle, y: 6 }, expected: colors[0] },
```

Comment:
Why this change? I think this makes the test stricter. Hopefully that doesn't make it fail on any devices but it's really hard to predict with the anisotropy stuff since its behavior is mostly unspecified.

Reply:
Reverted it for now. My thought was that given it's at anisotropy=1, that the results should be relatively deterministic, but we can revisit this later since I need to revisit this separately anyways.